Documentation/code-generation-instructions.md

```diff
@@ -55,7 +55,7 @@ RP is the resource provider's directory under SDKs, eg.: Compute
 To generate the `SdkInfo*.cs` file, please run the `generate.ps1` script
 
 ## Code generation artifacts
-If code is generated using `generate.ps1`, information related to the code generation gets logged in a `.txt` file under `src\SDKs\_metadata` eg.: [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/_metadata/compute_resource-manager.txt)
+If code is generated using `generate.ps1`, information related to the code generation gets logged in a `.txt` file under `sdk\_metadata` eg.: [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/_metadata/compute_resource-manager.txt)
```

Comment:
Might as well move the example file to `sdk\` and update the url so it makes more sense..